R/data.R

```diff
@@ -27,7 +27,7 @@
 
 #' @title Adult Seeds
 #' @description adult fish for the initial 5 years of the simulations derived
-#' from average escapement estimates from 2013 to 2017 (Azat 2019).
+#' from average escapement estimates from 2013 to 2017 \href{https://dsm-docs.s3.us-west-2.amazonaws.com/Azat+2019.pdf}{Azat 2019}.
```

Comment:
New issue: we need the adult seeds for the other runs. plz use this [grandtab data](https://raw.githubusercontent.com/CVPIA-OSC/DSMCalibrationData/main/data-raw/Grandtab_Modified.csv) to calculated the mean escapement 2013-2017 for the other runs. You can look at [this script](https://github.com/CVPIA-OSC/DSMCalibrationData/blob/add-year/data-raw/grandtab.R) to see what watersheds need an estimate for each run.